src/Microsoft.Build.Sql/README.md

```diff
@@ -33,3 +35,36 @@ In Visual Studio 2019 (or newer), .NET Core targets can be used by default. Add
 <NetCoreBuild>True</NetCoreBuild>
 ```
 and the project can be built directly in VS 2019 or `dotnet build` from command line.
+
+## Resolving build errors/warnings
+Depending on if your database project originated from SSDT or ADS, you may encounter build errors or warnings like these:
```

Comment:
might be nice to make "ADS" a link to the ADS repo in case anyone doesn't know what it is